### ISSUE TITLE
wr.s3.read_parquet chokes on _SUCCESS files

### ISSUE THREAD

**Issue by erikcw** [awslabs/aws-data-wrangler#326]
A `ArrowInvalid: Parquet file size is 0 bytes` exception is thrown when `wr.s3.read_parquet(location)` contains a `_SUCCESS` tombstone file.  This is problematic because Spark writes these _SUCCESS files automatically.  `read_parquet` should ignore them.

**Comment by igorborgest** [comment]
Hi @erikcw! Thanks for reaching out.

Now `_SUCCESS` files will be ignored by default.
In addition I've also implemented this new `path_ignore_suffix` argument  where you can pass a list of suffixes to be ignored.

Could you test our dev branch and check if it addresses your use case before the official release?
The intention is to release it in the version `1.7.0` later this week.

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released in [1.7.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.7.0)!

**Comment by igorborgest** [comment]
Hi @erikcw !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
